### PR TITLE
Add rpc cmd to return currently set fee increment

### PR DESCRIPTION
**Pull request by alexlyp**
This will provide a way in which users can see what their fee is currently set to while creating transactions.   
 dcrctl --wallet getwalletfee 
will return w.FeeIncrement value

### CODE REVIEW

**Comment by alexlyp** [comment]
I've added proper Get/get and Set/set functions for mutex safe access to feeIncrement (which is now unexported to enforce func usage)

File: wallet/wallet.go [hunk 1]
```diff
@@ -257,6 +259,36 @@ func (w *Wallet) SetStakeDifficulty(sdi *StakeDifficultyInfo) {
 	w.CurrentStakeDiff = sdi
 }
 
+// GetFeeIncrement is used to get the current feeIncrement for the wallet.
+// Uses non-exported thread safe feeIncrement func.
+func (w *Wallet) GetFeeIncrement() dcrutil.Amount {
```
Comment (jrick) [inline wallet/wallet.go]: Idiomatic go does not use Get for getters, just the field name.

File: wallet/wallet.go [hunk 2]
```diff
@@ -257,6 +259,36 @@ func (w *Wallet) SetStakeDifficulty(sdi *StakeDifficultyInfo) {
...
+	w.feeIncrementLock.Lock()
+	defer w.feeIncrementLock.Unlock()
+
+	return w.getFeeIncrement()
+}
+
+// feeIncrement is the non-exported mutex safe func to return current
+// w.FeeIncrement
+func (w *Wallet) getFeeIncrement() dcrutil.Amount {
```
Comment (jrick) [inline wallet/wallet.go]: This is unnecessary and should be removed.  The comment is also incorrect (it is NOT concurrent safe), but that won't matter after it's removed.

File: wallet/wallet.go [hunk 3]
```diff
@@ -257,6 +259,36 @@ func (w *Wallet) SetStakeDifficulty(sdi *StakeDifficultyInfo) {
...
+func (w *Wallet) SetFeeIncrement(fee dcrutil.Amount) {
+	w.feeIncrementLock.Lock()
+	defer w.feeIncrementLock.Unlock()
+
+	w.setFeeIncrement(fee)
+}
+
+// setFeeIncrement is the non-exported mutex safe func to set the current
+// w.FeeIncrement
+func (w *Wallet) setFeeIncrement(fee dcrutil.Amount) {
```
Comment (jrick) [inline wallet/wallet.go]: Unnecessary

File: wallet/wallet.go [hunk 4]
```diff
@@ -257,6 +259,36 @@ func (w *Wallet) SetStakeDifficulty(sdi *StakeDifficultyInfo) {
...
+	return w.feeIncrement
+}
+
+// SetFeeIncrement is used to set the current w.FeeIncrement for the wallet.
+// Uses non-exported mutex safe setFeeIncrement func
+func (w *Wallet) SetFeeIncrement(fee dcrutil.Amount) {
+	w.feeIncrementLock.Lock()
+	defer w.feeIncrementLock.Unlock()
+
+	w.setFeeIncrement(fee)
```
Comment (jrick) [inline wallet/wallet.go]:
```
w.feeIncrementLock.Lock()
w.feeIncrement = fee
w.feeIncrementLock.Unlock()
```

File: wallet/wallet.go [hunk 5]
```diff
@@ -257,6 +259,36 @@ func (w *Wallet) SetStakeDifficulty(sdi *StakeDifficultyInfo) {
 	w.CurrentStakeDiff = sdi
 }
 
+// GetFeeIncrement is used to get the current feeIncrement for the wallet.
+// Uses non-exported thread safe feeIncrement func.
+func (w *Wallet) GetFeeIncrement() dcrutil.Amount {
+	w.feeIncrementLock.Lock()
+	defer w.feeIncrementLock.Unlock()
+
+	return w.getFeeIncrement()
```
Comment (jrick) [inline wallet/wallet.go]: Also, no need for defer here.  See comment on the setter.

**Pull request merged**